Tita-Clone/Tita-Clone/Controllers/LoginViewController.swift

```diff
@@ -9,28 +9,137 @@ import UIKit
+        $0.textField.placeholder = "비밀번호"
+        $0.textField.isSecureTextEntry = true
+        $0.button.setTitle("비밀번호를 잊으셨나요?", for: .normal)
+        $0.button.setUnderline()
+        $0.button.addTarget(self, action: #selector(tapForgetPw), for: .touchUpInside)
+    }
     
+    private let enterButton = UIButton().then {
+        $0.setTitle("입장하기", for: .normal)
+        $0.setTitleColor(.black, for: .normal)
+        $0.dynamicFont(fontSize: 18, currentFontName: "NotoSans-Bold")
+        $0.backgroundColor = .rgb(red: 255, green: 251, blue: 235)
+        $0.layer.cornerRadius = 10
+        $0.layer.applySketchShadow(color: .black, alpha: 0.25, x: 2, y: 2, blur: 10, spread: 0)
+        $0.clipsToBounds = false
+        $0.addTarget(self, action: #selector(tapEnter), for: .touchUpInside)
+    }
+    
+    private let signUpButton = UIButton().then {
+        $0.setTitle("혹시 아직 회원이 아니신가요?", for: .normal)
+        $0.setTitleColor(.black, for: .normal)
+        $0.dynamicFont(fontSize: 12, currentFontName: "NotoSans-Regular")
+        $0.setUnderline()
+        $0.addTarget(self, action: #selector(tapSignUp), for: .touchUpInside)
+    }
     //MARK: - Lifecycle
     override func viewDidLoad() {
         super.viewDidLoad()
         configureUI()
+        keyboardSetting()
+    }
+
+    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?){
+         self.view.endEditing(true)
+   }
+
+    //MARK: - Keybord Setting
+    func keyboardSetting() {
+        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillShow(_:)), name: UIResponder.keyboardWillShowNotification, object: nil)
+        
+        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillHide(_:)), name: UIResponder.keyboardWillHideNotification, object: nil)
     }
     
     //MARK: - Selectors
+    @objc
+    func keyboardWillShow(_ sender: Notification) {
+        self.view.frame.origin.y = -(self.view.frame.height/5.5)
+        }
+    
+    @objc
+    func keyboardWillHide(_ sender: Notification) {
+        self.view.frame.origin.y = 0
+    }
+    
+    @objc
+    func tapForgetId() {
+        print("id 찾기")
+    }
+
+    @objc
+    func tapForgetPw() {
+        print("pw 찾기")
+    }
+    
+    @objc
+    func tapEnter() {
+        print("입장")
+    }
+    
+    @objc
+    func tapSignUp() {
+        print("회원가입")
+    }
     
     //MARK: - Helpers
     private func configureUI(){
         view.backgroundColor = .white
         addView()
         cornerRadius()
         location()
+        idContainerSetting()
+        pwContainerSetting()
     }
     
     // MARK: - Add View
     private func addView(){
-        
+        view.addSubview(backImg)
+        view.addSubview(logoImg)
+        view.addSubview(welcomeLabel)
+        view.addSubview(idContainer)
+        view.addSubview(pwContainer)
+        view.addSubview(enterButton)
+        view.addSubview(signUpButton)
```

Comment:
```suggestion
        [backImg, logoImg, welcomeLabel, idContainer, pwContainer, enterButton, signUpButton].forEach { view.addSubview($0) }
```
이렇게 `forEach`문을 사용하면 property들을 한 줄로 `addSubview`할 수 있습니다.